src/commands/CleanCommand.js

```diff
@@ -1,11 +1,26 @@
-import async from "async";
-import Command from "../Command";
 import FileSystemUtilities from "../FileSystemUtilities";
 import PromptUtilities from "../PromptUtilities";
+import Command from "../Command";
+import async from "async";
```

Comment:
Please don't reorder the imports for cosmetic reasons, it's a historical artifact that I'd rather not propagate.